libseccomp/src/notify.rs

```diff
@@ -160,6 +170,33 @@ impl ScmpNotifReq {
     ///
     /// If an issue is encountered getting a notification request,
     /// an error will be returned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use libseccomp::*;
+    /// # use libc::fork;
+    /// # use std::process::{exit, id};
+    /// // Get the current process ID
+    /// let parent_pid = id();
```

Comment:
`std::prcocess::id()` is better because our users cannot read the `use std::process::{exit, id};` line in the doc.